### DISCUSSION TITLE
Please fix the 2022 battlepass, add "Arcana Token"s as rewards

### DISCUSSION THREAD

**Discussion by SirStreetguru** [ValveSoftware/Dota2-Gameplay#2673] [Ideas]
I don't see much value in buying a level 100 battlepass this year, there isn't a single large reward for spending $45???

There's a simple fix for this, at levels 50/100/200/300 add rewards called "Arcana Token" and "Persona Token" letting people pick which one they want at each point, so long as they alternate their choice. Replacing the existing persona/arcana slots naturally. 

The tokens never expire and can be used to trade in for any persona or arcana past or future. 

This actually grants value to the battlepass and lets people who only have a level 1 pass grind up a real reward or 2 if they so choose. 

Also what happened to the old forums? They're just gone forever? 

*This type of token system could of course apply to every other type of item in the game as well. Could have common set tokens/uncommon set tokens, ect. As additional battlepass rewards/ or quest rewards/completion rewards.

**Comment by ArtyomKamnev** [comment]
So you are expecting to get personas and arcanas for every 10$ you pay

**Reply by SirStreetguru** [reply]
I can already farm the CM persona with a level 1 pass for $10. 

It doesn't have to be at 50/100, maybe that could be a bonus for buying a level 100 pass to start with.

But it would still cost $30, and $22 each. The cost seems just fine given that it's a once a year deal at that point, so unless valve wants to be extra greedy, giving people a once a year deal for an arcana or persona of their choice seems fine to me.
